Makefile

```
@@ -1,7 +1,7 @@
 SHELL := /bin/bash
 export GO111MODULE ?= on
 export VERSION ?= v1.1.0
-export ECE_VERSION ?= 2.7.0-ms47
+export ECE_VERSION ?= 2.8.0
```

I believe ECE 2.8.0 is not scheduled to go out until `MS-49` so we should tag this with the `-ms48` suffix.
```suggestion
export ECE_VERSION ?= 2.8.0-ms48
```